services/web/server/src/simcore_service_webserver/meta_modeling_results.py

```diff
@@ -0,0 +1,77 @@
+""" Access to the to projects module
+
+    - Adds a middleware to intercept /projects/* requests
+    - Implements a MetaProjectRunPolicy policy (see director_v2_abc.py) to define how meta-projects run
+
+"""
+
+
+import logging
+from typing import Any, Dict
+
+from models_library.projects_nodes import Outputs
+from models_library.projects_nodes_io import NodeIDStr
+from pydantic import BaseModel, Field, conint
+
+log = logging.getLogger(__name__)
+
+
+class ExtractedResults(BaseModel):
+    progress: Dict[NodeIDStr, conint(ge=0, le=100)] = Field(
+        ..., description="Progress in each computational node"
+    )
+    labels: Dict[NodeIDStr, str] = Field(
+        ..., description="Maps captured node with a label"
+    )
+    values: Dict[NodeIDStr, Outputs] = Field(
+        ..., description="Captured outputs per node"
+    )
```

Comment:
could you please provide an example here, to make it more clear what it produces? Will help in the future.